### ISSUE TITLE
Enhancement suggestion: Use icon of APP for Entry 

### ISSUE THREAD

**Issue by Opensourcecommunitydevelopment** [PhilippC/keepass2android#68]
like plugin favicon downloader does this in desktop for uri, keepass2android could do this for entrys by app uri. making entry list more intuitive and beautiful.

**Comment by ikellenberger** [comment]
Both could be used:
- icon of app
- favicon of URL

Thnx!

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
yes, both sources would be nice. Appicon as primary and favicon as fallback if url is present.

**Comment by tom8184** [comment]
Hallo,
Gibt es etwas neues zu diesem Thema?
Ich würde es sehr begrüßen, da es die Orientierung deutlich vereinfachen würde.

**Comment by LuisAPI** [comment]
Would be really helpful!

**Comment by iluis-ud** [comment]
I think the fact that there's been multiple duplicates of this closed shows that this deserves to be more than just a Backlog. \:)
